README.md

```diff
@@ -1,13 +1,12 @@
-# Origami asset loader [![NPM version](https://img.shields.io/npm/v/@financial-times/health-check.svg)](https://www.npmjs.com/package/@financial-times/health-check) [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
-
+# o-assets [![MIT licensed](https://img.shields.io/badge/license-MIT-blue.svg)](#licence)
```

Comment:
🙌

Reply:
![tooltip over this line of code showing: origami-component/readme-has-name The main heading in the README must be the component's name as defined in the bower.json. expected "o-assets", got "some asset loader"](https://user-images.githubusercontent.com/178266/83515901-a27f8480-a4c5-11ea-8d04-0b41a3498c9a.png)

:D

Reply:
![tooltip over this line of code reading: origami-component/readme-has-name The main heading in the README must be the component's name as defined in the bower.json. expected "o-assets", got "Origami asset loader"](https://user-images.githubusercontent.com/178266/83515987-c9d65180-a4c5-11ea-8e11-d56559331c0c.png)
:D